services/skus/handler/handler.go

```diff
@@ -117,6 +120,29 @@ func (h *Order) CreateNew(w http.ResponseWriter, r *http.Request) *handlers.AppE
 	return handlers.RenderContent(ctx, result, w, http.StatusCreated)
 }
 
+func (h *Order) Cancel(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+	ctx := r.Context()
+
+	orderID, err := uuid.FromString(chi.URLParamFromCtx(ctx, "orderID"))
+	if err != nil {
+		return handlers.ValidationError("request", map[string]interface{}{"orderID": err.Error()})
+	}
+
+	lg := logging.Logger(ctx, "skus").With().Str("func", "CancelOrderNew").Logger()
```

Comment:
The reason it's `CancelOrderNew` is that we don't have yet an established pattern for methods of handlers. I could not call it `CancelOrder`, because there is already one – the legacy "controller".

It also does not make sense to call it `Cancel` in the current form – because the logger to which this sublogger is attached is too generic – it's `skus`. So `func` with value `Cancel` is not specific enough.

We need to come up with something like below for methods of handlers:

```golang
lg := logging.Logger(ctx, "skus").With().Str("handler", "Order").Str("func", "Cancel").Logger()
```

Since we're not there yet, I will leave it for now because it does the job – it identifies uniquely and unambiguously the place where it's created and used.